doc/api/cluster.markdown

```diff
@@ -24,7 +24,7 @@ all share server ports.
       });
     } else {
       // Workers can share any TCP connection
-      // In this case its a HTTP server
+      // In this case it is an HTTP server
```

Comment:
@joyent/node-coreteam Could a native English speaker please confirm the usage of "an" instead of "a" here? Thank you!